src/Crocodealer/Command/Sync.hs

```diff
@@ -0,0 +1,63 @@
+module Crocodealer.Command.Sync
+       ( sync
+       ) where
+
+import Network.HTTP.Simple (getResponseBody, getResponseStatus, httpBS, parseRequest)
+import Network.HTTP.Types.Status (Status (..))
+
+import Crocodealer.ColorTerminal (errorMessage, infoMessage, warningMessage)
+import Crocodealer.Core.GitHub (FileName (..), RepoName (..), UserName (..))
+
```

Comment:
According to our style guide, imports should be separated by two empty lines from the code. This makes code a bit more readable :slightly_smiling_face: 

```suggestion


```